src/plugins/intel_cpu/src/nodes/fake_quantize.cpp

```diff
@@ -1534,12 +1570,12 @@ void FakeQuantize::executeReference() {
             float src_val = src[src_off];
 
             int wei_idx = getAxis() == 0 ? n : c;
-            float cl = crop_low[wei_idx];
-            float ch = crop_high[wei_idx];
-            float isc = input_scale[wei_idx];
-            float ish = input_shift[wei_idx];
-            float osc = output_scale[wei_idx];
-            float osh = output_shift[wei_idx];
+            float cl = cropLow[wei_idx * broadcastFactor[CROP_LOW]];
+            float ch = cropHigh[wei_idx * broadcastFactor[CROP_HIGH]];
+            float isc = inputScale[wei_idx * broadcastFactor[INPUT_SCALE]];
+            float ish = inputShift[wei_idx * broadcastFactor[INPUT_SHIFT]];
+            float osc = outputScale[wei_idx * broadcastFactor[OUTPUT_SCALE]];
+            float osh = outputShift[wei_idx * broadcastFactor[OUTPUT_SHIFT]];
```

Comment:
I'm wondering, may be it would be better to use a bitset, something like this:
```cpp
            float cl =  broadcasted[CROP_LOW] ? cropLow[wei_idx] : cropLow.front();
            ...
```
also

```cpp
    arg.crop_low =  broadcasted[CROP_LOW] ? &cropLow[c] : &cropLow[0];
```
And get rid on the broadcastFactor at all. I would agree to keep this additional vector if it contained values different from 0 and 1, but this is not the case.

Reply:
fixed, thanks!